routes/users.js

```diff
@@ -10,65 +10,81 @@ router.use(bypassIfAuthorised)
 
 // keep static route at top
 router.get('/register', (req, res) => {
-    res.render('users/register', locals({firstName: 'placeholder'}))
+    res.render('users/register')
 })
 
 router.get('/login', (req, res) => {
     res.render('users/login')
 })
 
 router.post('/login', async (req, res) => {
-    const { email, password } = req.body;
+    const { email, password } = req.body
     try {
         const dbUser = await User.findAll({
             where: {
-              email: email
+                email: email,
             },
-          });
-          if (!dbUser[0]?.dataValues) {
-            res.render('users/login', locals({ email }, 'Incorrect email or password'))
+        })
+        if (!dbUser[0]?.dataValues) {
+            res.render(
+                'users/login',
+                locals(
+                    { email, authorised: req.session?.authorised },
+                    'Incorrect email or password.'
+                )
+            )
             return
-          }
+        }
 
-            const { password: dbPassword } = dbUser[0]?.dataValues;
-            const match = await comparePassword(password, dbPassword);
-            console.log('match', match);
+        const { password: dbPassword } = dbUser[0]?.dataValues
+        const match = await comparePassword(password, dbPassword)
+        console.log('match', match)
 
-            if (match) {
-                    console.log(req.sessionID);
-                    req.session.authorised = true;
-                    req.session.user = dbUser[0]?.dataValues;
-                    console.log(dbUser[0]?.dataValues);
-                res.redirect('/account/index')
-                return
-            } else {
-                res.render('users/login', locals({ email }, 'Forgot password?'))
-                return
-            }
+        if (match) {
+            console.log(req.sessionID)
+            req.session.authorised = true
+            req.session.user = dbUser[0]?.dataValues
+            console.log(dbUser[0]?.dataValues)
+            res.redirect('/account/index')
+            return
+        } else {
+            res.render(
+                'users/login',
+                locals(
+                    { email, authorised: req.session?.authorised },
+                    'Incorrect password.'
```

Comment:
Generally you don't want to specifically say that the password is wrong - keep it generic, see owasp guidelines here https://cheatsheetseries.owasp.org/cheatsheets/Authentication_Cheat_Sheet.html#authentication-responses